examples/make_eeg_bids.py

```diff
@@ -0,0 +1,199 @@
+"""
+=========================
+Use MNE-BIDS for EEG Data
+=========================
+
+In this example, we use MNE-BIDS to create a BIDS-compatible directory of EEG
+data. Specifically, we will follow these steps:
+
+#. Download some EEG data from the
+`PhysioBank database <https://physionet.org/physiobank/database>`_.
+
+#. Load the data, extract information, and save in a new BIDS directory
+
+#. Check the result and compare it with the standard
+
+"""
+
+# Authors: Stefan Appelhoff <stefan.appelhoff@mailbox.org>
+# License: BSD (3-clause)
+
+###############################################################################
+# We are importing everything we need for this example:
+import os
+import shutil as sh
+
+from mne.datasets import eegbci
+from mne.io import read_raw_edf
+
+from mne_bids import raw_to_bids
+from mne_bids.utils import print_dir_tree
+
+###############################################################################
+# Step 1: Download the data
+# -------------------------
+#
+# First, we need some data to work with. We will use the
+# `EEG Motor Movement/Imagery Dataset <https://doi.org/10.13026/C28G6P>`_
+# available on the PhysioBank database.
+#
+# The data consists of 109 volunteers performing 14 experimental runs each. For
+# each subject, there were two baseline tasks (1. eyes open, 2. eyes closed) as
+# well as four different motor imagery tasks. For the present example, we will
+# show how to format the data for two subjects and selected tasks to comply
+# with the Brain Imaging Data Structure
+# (`BIDS <http://bids.neuroimaging.io/>`).
+#
+# Conveniently, there is already a data loading function available with
+# MNE-Python:
+
+# Make a directory to save the data to
+home = os.path.expanduser('~')
+mne_dir = os.path.join(home, 'mne_data')
+if not os.path.exists(mne_dir):
+    os.makedirs(mne_dir)
```

Comment:
why is this necessary?

Reply:
Because I am using `print_dir_tree` on that directory later ... and if I didn't introduce it here, the path would come "out of nowhere" because `eegbci.load_data()` would have picked the default path. I added it for a better flow of the example

Reply:
can you share how the link of the built example? So I can understand better

Reply:
okay I see what you mean, but `eegbci.load_data` returns the `path`. Can't we use that?

Reply:
so what I'm concerned about is that this path may not always be `mne_data`. See here: https://github.com/mne-tools/mne-python/blob/master/mne/datasets/eegbci/eegbci.py#L27-L32. It depends on your MNE config and I think it's quite common to not store your data in the home directory.

Reply:
> As for a link ... now it finally pays off that we started using circleci

yes totally !